vast/integration/vast_integration_suite.yaml

```diff
@@ -735,6 +735,7 @@ tests:
     tags: [pipelines]
     steps:
       - command: exec 'from file @./data/suricata/eve.json read suricata | write json to stdout'
+      - command: exec 'from file @./data/suricata/eve.json read json --schema=suricata.alert --no-infer | write json to stdout'
```

Comment:
Does this test actually test that this works? The output doesn't include `"suricata.alert"`. How about something like `put #schema`?

Reply:
I think we need a test where we assign the table slice simply a fixed name, independent of existing schemas.

Reply:
And that name would be good to have in the test output.

Reply:
That appears to be independent of the feature added by this PR. I'll add a note to a related issue to add such a test.